hack/test-cmd.sh

```diff
@@ -699,7 +699,7 @@ __EOF__
   # Command: autoscale rc "frontend"
   kubectl autoscale -f examples/guestbook/frontend-controller.yaml --save-config "${kube_flags[@]}" --max=2
   # Post-Condition: hpa "frontend" has configuration annotation
-  [[ "$(kubectl get hpa frontend -o yaml "${kube_flags[@]}" | grep kubectl.kubernetes.io/last-applied-configuration)" ]]
```

Comment:
The argument we originally made is that it's left to right familiar for users:

pods
pods.extensions
pods.extensions.k8s.io (eventually)

Putting extension first when the vast majority of the time the
resource would be first felt wrong.